float32 zero
float32 one
float32 two
float32 three
float32 four
float32 five
